fixtures/public-dep-public-pkg/packages/public-pkg/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "@fixture/public-pkg",
+    "version": "0.0.1",
+    "dependencies": {
+        "@fixture/public-dep": "^0.0.1"
+    }
+}
```

Comment:
Love all these test fixtures. Is `@fixture` a reserved namespace or just something you chose to use just for testing?

Reply:
It's just something I made up fro testing... nothing actually gets published as part of the test tests.